doc/05-Upgrading.md

```diff
@@ -11,9 +11,9 @@ If you are upgrading across multiple versions, make sure to follow the steps for
 
 **Breaking Changes**
 
-The JSON/CSV-export for Hosts and Services now supports filtering for specific columns with the `?columns` parameter.
-The exported file will only contain the listed columns and the id of the Host/Service, if the parameter is not set, all columns will be included.
-If a relation is entirely empty and would result in an empty JSON-object, the JSON-export will not create an object for it at all.
+* The JSON/CSV-export for Hosts and Services now supports filtering for specific columns with the `?columns` parameter.
+  * The exported file will only contain the listed columns of the Host/Service, if the parameter is not set, all columns will be included.
+  * If a relation is entirely empty and would result in an empty JSON-object, the JSON-export will not create an object for it at all.
```

Comment:
That should be its own first level list item. It's not dependent on `?columns` after all. Also, please give an example, I'm not sure everyone is familiar with the term *relation*.